src/olympia/amo/tests/__init__.py

```diff
@@ -806,6 +821,12 @@ def empty_index(cls, index):
             body={"query": {"match_all": {}}}
         )
 
+# We need ES indexes aliases to match prod behaviour, but also we need the
+# names need to stay consistent during the whole test run, so we generate them
+# at import time.
+ESTestCase.indexes_names = {key: timestamp_index(value)
+                            for key, value in settings.ES_INDEXES.items()}
+
```

Comment:
I moved it up to the class declaration. It works in tests because pytest initialises `ES_INDEXES` before the tests actually run - it's not using dynamic setting overriding.